README.md

```diff
@@ -1,8 +1,122 @@
-# apps
-Proof of concept applications that use the Force Move Game framework.
-
-### Setting up development environment and running the app
-* In the top directory, run `yarn install`.
-* In the top directory, run `npx lerna bootstrap`.
-* Run the wallet via `yarn start` in the `wallet` package directory
-* Run the app via `yarn start` in the app package directory.
+<h1 align="center">
+<div><img src="./orange_fireball.svg"> </div>
+Magmo Apps
+</h1>
+Welcome to the Magmo mono-repo, home of several proof-of-concept applications built on our state channel protocols.
+
+## For more information
+On our [website](https://magmo.com) you will find links to our whitepapers and contact information. Whether you simply want to try
+out our apps, or get involved more deeply we would love to hear your thoughts. Deployed versions of our games may be accessed with these links:
+
+* [Rock Paper Scissors](https://demo.magmo.com) (RPS)
+* [Tic Tac Toe](https://ttt.magmo.com) (TTT)
+
+## Setting up development environment and running a game application
+You will need `yarn` installed (see [here](https://yarnpkg.com/lang/en/docs/install/) for instructions). After cloning the code, 
+1. In the top directory, run `yarn install`.
+2. Run `npx lerna bootstrap`.
+3. Add `.env` files to the root directory of the `wallet` package, and to the root directory of the relevant app directory. Refer to `.env.example` which includes an example of the required variables.
+
+4. Start [ganache](https://truffleframework.com/ganache) by running `yarn ganache:start` .
+5. (In a new shell) Run the wallet via `yarn start` in the `wallet` package directory
+6. (In a new shell) Run a game (either RPS or TTT) via `yarn start` in the relevant package directory.
+7. Add [MetaMask](https://metamask.io/) to your browser, and point it to `localhost:3001` to view the application. You will need to import one of our (testnet-only) seed accounts into metamask to have funds to transact.
+    1. Open the metamask browser extension
+    2. Click on the account icon (circle in the top right)
+    3. Select "Import"
+    4. Paste in one of [these secret keys](https://github.com/magmo/devtools/blob/master/utils/startGanache.js).
+
+You may visit the app in two different browsers in order to play against yourself. We use [Redux DevTools](https://github.com/reduxjs/redux-devtools) to develop and test our apps.
+
+#### To run storybook
+
+We use [Storybook](https://storybook.js.org/) to view our react components during development. You can start Storybook by running:
```

Comment:
@geoknee This has to be run inside a package directory, right? If so, we might want to specify that.